config/formats.ts

```diff
@@ -565,8 +561,10 @@ const Formats = [
 				if (restrictedPokemon.includes(species.name)) {
 					return [`${species.name} is not allowed to hold ${item.name}.`];
 				}
-				if (itemTable[item.id]) return ["You are limited to one of each mega stone.", "(You have more than one " + item.name + ")"];
-				itemTable[item.id] = true;
+				if (itemTable.has(item.id)) {
+					return ["You are limited to one of each mega stone.", "(You have more than one " + item.name + ")"];
```

Comment:
```suggestion
					return [
						`You are limited to one of each mega stone.`,
						`(You have more than one ${item.name})`
					];
```

(I don't go out of my way to change old strings into template strings, but if you're going to be modifying them anyway, I think that's a great time to do it.)